js/yoast-compat.js

```diff
@@ -99,6 +117,12 @@ jQuery(function($){
 								}
 								newHTML = newHTML.prop( 'outerHTML' );
 								break;
+							case 'SiteOrigin_Widget_Button_Widget':
+								var hrefSeparator = widgetInstance.url.includes( '://' ) === true ? '' : '#';
```

Comment:
Is `=== true` necessary? This function returns either `true` or `false`, so it could just be `widgetInstance.url.includes( '://' ) ? '' : '#'`

Reply:
I was basing it on [this](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#equality). If you would like me to change it, let me know and I'll change it - I personally don't mind.

Reply:
Yeah, as a general rule it's better to use `===`, but in this case, even using `==` would be incorrect. Functions that return booleans, don't need to be compared to a boolean except in a few special cases. Like if `includes` returned `false` if the string didn't exist and an integer position if it did exist, where `0` was a valid position. Then `!== false` would be okay.

Unnecessary code makes things harder to read.

Reply:
That's understandable and makes sense. I'll make the change.